### PR TITLE
feature/update-actions-to-work-with-python3

### PR DESCRIPTION
**Pull request by jschoewe**
Updated all actions to work with Python 3 and fixed bugs in vm_snapshots_delete and get_vmconsole_urls actions.

This PR is a fix for Issue #79

### CODE REVIEW

**Comment by jschoewe** [comment]
@shivani-orch please see the CHANGELOG notes regarding the vm_snapshots_delete action as it is currently broken

File: pack.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ ref: vsphere
 name: vsphere
 description: VMware vSphere
 stackstorm_version: ">=2.9.0"
-version: 1.0.1
+version: 1.0.2
```
Comment (bishopbm1) [inline pack.yaml]: lets make this version `1.1.0`

File: requirements.txt [hunk 2]
```diff
@@ -1 +1 @@
-pyvmomi==5.5.0
+pyvmomi
```
Comment (bishopbm1) [inline requirements.txt]: pin to the latest version

**Review by bishopbm1 — COMMENTED** [review]
A few small changes

**Review by bishopbm1 — APPROVED** [review]
LGTM!

**Pull request merged**